poglink/bot.py

```diff
@@ -15,7 +15,7 @@ def __init__(
         bans_channel_id=None,
         polling_delay=None,
         allowed_roles=None,
-        rates_url=None,
+        rates_urls=None,
```

Comment:
I'm a little concerned this wasn't acting broken before, but it could be fleshed out better in tests if there's something we're missing.

Reply:
Is it passed kwargs from [setup_config](https://github.com/FM-17/poglink/blob/5eed54d5c3f0b5fe6e8edb2fba6a7e346cf084d6/poglink/config.py#L35)? If so, wouldn't the [rates_urls](https://github.com/FM-17/poglink/blob/5eed54d5c3f0b5fe6e8edb2fba6a7e346cf084d6/poglink/config.py#L15) default value initialization in config.py be used? Resulting in the non-plural `rate_url` being ignored?

Reply:
Ah you hit the nail on the head. That must be what was going on.

If we were using `vyper-config` for this, we'd be able to actually introspect the values a little better, but it looks like it was a case of two wrongs making a right maybe (or maybe there was a flaw in the test logic still)

Reply:
Yeah for sure. `vyper-config` might be a good fit. It would even simplify the Discord-based configuration interface I was hoping to add later on, since you can just use [`v.set`](https://github.com/alexferl/vyper#setting-overrides) to set any config pulled from user input.

Reply:
#41